doc/MSB.md

```diff
@@ -282,11 +282,13 @@ All configuration files use _key-value pair_ structure.
 ### Description of MSB configuration fields
 Service details section describes microservice parameters.
 
-- `name ` – microservice name
-- `version` – microservice version
-- `instanceId` – unique microservice instance id
+- `name ` – microservice name. All run instances of the same microservice must have the same name. 
+- `version` – microservice version. At the moment Msb-java doesn't handle the value.
+- `instanceId` – unique microservice instance id. All run instances of the same microservice should have different instance ids.
```

Comment:
Run -> Running?

Maybe let's explicitly specify that after microservice restart the value should be different. Also let's mention how this value is used (to construct reply topic name)